src/stores/room-list/algorithms/list-ordering/ImportanceAlgorithm.ts

```diff
@@ -257,6 +262,7 @@ export class ImportanceAlgorithm extends OrderingAlgorithm {
 
             if (indices[lastCat] > indices[thisCat]) {
                 // "should never happen" disclaimer goes here
+                // eslint-disable-next-line max-len
```

Comment:
why does this one get a max-len exclusion but the others don't?

Reply:
This one no longer does